### PR TITLE
Update threejs version and fix isosurface color bug

### PR DESCRIPTION
**Pull request by toloudis**
Update threejs version to latest. 

Some small code fixups based on deprecation warnings:  matrix inverse has changed, and also they got rid of the Geometry class which caused one mesh data-filling function to be somewhat rewritten.

In testing the update, I found and fixed the following bug:

When updating a surface's isovalue, we destroy and re-create the surface, and to set the new surface to the correct color, the code was looking at the previous threejs object to find out what color it was.  But when the isovalue is changed to a value that doesn't generate any triangles, then there's no mesh in which to look up the color for the next change.

The fix is to just store the current colors (and opacities) in their own array.

### CODE REVIEW

**Comment by toloudis** [comment]
Also hopefully this will be a better version of #13 and we can get rid of that one.

File: src/vr/vrObjectControls.js [hunk 1]
```diff
@@ -204,7 +204,7 @@ export class vrObjectControls {
       v1 = v1.normalize();
 
       var mio = new Matrix4();
-      mio.getInverse(obj3d.matrixWorld);
+      mio.copy(obj3d.matrixWorld).invert();
```
Comment (meganrm) [inline src/vr/vrObjectControls.js:207]: is this a file you wrote or is it copied from some library?

File: src/MeshVolume.js [hunk 2]
```diff
@@ -147,30 +155,11 @@ export default class MeshVolume {
...
-    } else {
-      this.meshrep[channel].traverse(function(child) {
-        if (child instanceof Mesh) {
-          opacity = child.material.opacity;
-          color = child.material.color;
-        }
-      });
-    }
+    let opacity = this.channel_opacities[channel];
+    let color = this.channel_colors[channel];
```
Comment (meganrm) [inline src/MeshVolume.js:159]: this is real nit but here and below you're using snake_case in javascript files. In other repos I think our eslint rules are set up to enforce camelCase but I guess not here?

**Review by meganrm — APPROVED** [review]

File: src/vr/vrObjectControls.js (hunk 1 above)
Comment (toloudis) [reply src/vr/vrObjectControls.js:207]: wrote

Comment (toloudis) [reply src/vr/vrObjectControls.js:207]: vr support in this lib is pretty much dead at the moment anyway

File: src/MeshVolume.js (hunk 2 above)
Comment (toloudis) [reply src/MeshVolume.js:159]: yup, I noticed this too, I've been in python land for too long. The code is due for a big typescript conversion at some point soon.  Probably at the time of the next big feature addition, I will take a pause and start adding conformance with our other more modern js/ts code.

**Review by blairlyons — APPROVED** [review]

**Pull request merged**